sanic_ext/extensions/openapi/openapi.py

```diff
@@ -98,8 +98,14 @@ def inner(func):
 
 
 def document(url: Union[str, ExternalDocumentation], description: str = None):
+    if isinstance(url, ExternalDocumentation):
+        _url = url.fields["url"]
+        description = url.fields["description"]
+    elif isinstance(url, str):
+        _url = url
```

Comment:
There's no else which could lead to variable not being declared.

Reply:
Thank you. The logic was improved.